ProteoformSuiteInternal/Component.cs

```diff
@@ -141,9 +151,9 @@ public Component()
 
 
         // METHODS
-        public void add_charge_state(List<string> charge_row, double correction)
+        public void add_charge_state(List<string> charge_row)
```

Comment:
Something we should talk about next week - I integrated calibration into PS so it writes a new excel with the calibrated value, don't need correction from .txt files anymore if we decide to keep it